CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_ad_reporting v1.13.0
+
+## Breaking Changes: Version Updates
+The following dependencies have been updated following upstream breaking changes. See the below release notes for more information.
+  - `dbt_apple_search_ads` ([v0.5.0](https://github.com/fivetran/dbt_apple_search_ads/releases/tag/v0.5.0))
+  - `dbt_microsoft_ads` ([v0.10.0](https://github.com/fivetran/dbt_microsoft_ads/releases/tag/v0.10.0))
+The `conversions` field now exists for `apple_search_ads`, using `tap_installs` as the titular field.
```

Comment:
I believe conversions were included already, we just didn't need to explicitly include them in the get_query `field_mapping` because they matched the default field name "conversions"

Reply:
ok gotcha. i might just make note that we switched the native field that feeds into it then